aws/rust-runtime/aws-types/src/lib.rs

```diff
@@ -4,6 +4,8 @@
  */
 
 pub mod build_metadata;
+pub mod environment;
```

Comment:
Should we make these doc hidden and name this something like `internal_env` to avoid requiring it to have a stable interface?